web/src/js/ducks/flows.js

```diff
@@ -117,6 +117,17 @@ export function update(flow, data) {
     return { type: REQUEST_ACTION }
 }
 
+export function update_content(flow, file) {
```

Comment:
yes of course, i just handled every where the response case, when everything is working i will add the stuff for response